scripts/save.sh

```diff
@@ -313,7 +315,7 @@ main() {
 			start_spinner "Saving..." "Tmux environment saved!"
 		fi
 		save_all
-		if show_output; then
+    if show_output; then
```

Comment:
Can we please use tabs for indentation?

I'm also not happy with tabs and would like to change to two-space indentation, but I need help from someone that's willing to do that for all the repos in `tmux-plugins` github org.